addons/base_import/static/src/xml/base_import.xml

```diff
@@ -63,6 +63,15 @@
                         Track history during import
                     </label>
                 </div>
+                <div title="If the model uses parent/child relations, computing the 
+                    parent / child relation occurs on every line, and lead to a slower import.
+                    Defering it can speed up import."
+                    class="oe_import_deferparentstore">
```

Comment:
maybe only shown in debug mode?

Reply:
Already done:
if (result.debug === false){
              this.$('.oe_import_tracking').hide();		              this.$('.oe_import_tracking').hide();
              this.$('.oe_import_deferparentstore').hide();
}

Reply:
👍